original/java/db.properties

```diff
@@ -0,0 +1,5 @@
+#mysql DB properties
```

Comment:
I think I missed something. Do you mean to change the file name to db.properties.default?